karma.conf.js

```diff
@@ -25,9 +52,20 @@ module.exports = function karmaConf(config) {
     preprocessors: {
       'tests/**/Test*.js': ['webpack', 'sourcemap'],
     },
-    reporters: ['mocha'],
-    port: 9876,
+    reporters,
     colors: true,
+    port: 9876,
+    plugins: [
+      'karma-chai',
```

Comment:
I think karma loads all of the plugins by default now so this shouldn't be needed/